src/components/shared/formItems/formHelpers.tsx

```diff
@@ -0,0 +1,107 @@
+import { PayoutMod } from 'models/mods'
+import { fromPermyriad } from 'utils/formatNumber'
+import { constants, utils } from 'ethers'
+import { BigNumber } from '@ethersproject/bignumber'
+
+import { stripCommas, formatWad, parseWad } from 'utils/formatNumber'
+import { amountSubFee, amountAddFee } from 'utils/math'
+
+// Get total percentages from a list of mods such as in Project->'Edit payouts'
+export function getTotalPercentage(mods: PayoutMod[] | undefined) {
+  return (
+    mods?.reduce(
+      (acc, curr) => acc + parseFloat(fromPermyriad(curr.percent ?? '0')),
+      0,
+    ) ?? 0
+  )
+}
+
+// Ensures value is greater than 0 and less than 100
+export function validatePercentage(percent: number | undefined) {
+  if (percent === undefined || percent === 0) return Promise.reject('Required')
+  else if (percent > 100) return Promise.reject('Invalid')
+  return Promise.resolve()
+}
+
+// Validates an eth address from a modal where the address
+// is being added to a set of addresses (such as payout or
+// reserved token receivers)
+// Ensures address is not null, 0, or the same as an already added address
+export function validateEthAddress(
+  address: string,
+  mods: PayoutMod[],
+  modalMode: string | undefined,
```

Comment:
**question**: I seem to remember we have a `modalMode` enum type somewhere. Can we reuse it here?